hack/update-generated-protobuf.sh

```diff
@@ -37,7 +37,13 @@ function prereqs() {
   KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
   KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
   DOCKER_MOUNT_ARGS=(
-    --volume "${REPO_DIR:-${KUBE_ROOT}}:/go/src/${KUBE_GO_PACKAGE}"
+    --volume "${REPO_DIR:-${KUBE_ROOT}}/cluster:/go/src/${KUBE_GO_PACKAGE}/cluster"
```

Comment:
Is it selinux or simply "container runs as root" ?

On Mon, May 9, 2016 at 11:45 PM, Maciej Szulik <notifications@github.com>
wrote:

> In hack/update-generated-protobuf.sh
> <https://github.com/kubernetes/kubernetes/pull/25354#discussion_r62619851>
> :
>
> > @@ -37,7 +37,13 @@ function prereqs() {
> >    KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
> >    KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
> >    DOCKER_MOUNT_ARGS=(
> > -    --volume "${REPO_DIR:-${KUBE_ROOT}}:/go/src/${KUBE_GO_PACKAGE}"
> > +    --volume "${REPO_DIR:-${KUBE_ROOT}}/cluster:/go/src/${KUBE_GO_PACKAGE}/cluster"
>
> In short, it's related to SELinux re-labeling happening to mount directory
> inside the container. All the directories/fies that are then created inside
> the container are owned by root, which causes problems @gmarek
> <https://github.com/gmarek> mentioned in his issue. The other possible
> approach is to build the binaries outside the container, then there's no
> problem (see my proposed solution for that: #25346
> <https://github.com/kubernetes/kubernetes/pull/25346>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25354/files/9a8e4f69cd707441ea9f7996a983f9e30a30a5ce#r62619851>
>